drivers/ieee802154/ieee802154_nrf5.c

```diff
@@ -183,6 +185,26 @@ static int nrf5_set_channel(struct device *dev, u16_t channel)
 	return 0;
 }
 
+static int nrf5_energy_scan_start(struct device *dev,
+				  u16_t duration,
+				  energy_scan_done_cb done_cb)
+{
+	int err = 0;
+
+	ARG_UNUSED(dev);
+	if (energy_scan_done == NULL) {
+		energy_scan_done = done_cb;
+		if (nrf_802154_energy_detection(duration * 1000) == false) {
```

Comment:
same

(it's a linux rule that zephyr follows btw)

Reply:
added empty line before if